src/components/constants/Items.ts

```diff
@@ -408,6 +408,226 @@ export class Items {
         "Town Map": {
             "effect": "Shows your position in the Pokemon World",
             "category": "Key"
-        }
+        },
+        "HM01": {
+            "effect": "Cut",
```

Comment:
All known moves up to the most current gen have no exception like that (some moves have hyphens), but if it does ever come along I could just have it like the Pokemon.ts file handles the names as arrays. As far as it seems the current Item class with a small revamp works fine for items in my opinion